distributed/scheduler.py

```diff
@@ -3808,10 +3809,6 @@ async def close(self, comm=None, fast=False, close_workers=False):
         for comm in self.client_comms.values():
             comm.abort()
 
-        await self.rpc.close()
-
-        self.status = Status.closed
-        self.stop()
```

Comment:
Why this change?

Reply:
this is performed by `super().close()` since there are no other things happening in between, letting the server close itself felt like the right thing to do to reduce complexity.